web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -13,7 +13,7 @@
               Embargo this Dandiset
               <v-tooltip
                 location="right"
-                max-width="25%"
+                max-width="400px"
```

Comment:
While this does fix the problem, it sets the max width of this tooltip to a fixed value, irrespective of the user's display size/resolution. I think the intent of the original `max-width=25%` was to have it expand up to 25% of the display; we can accomplish this by setting `width` instead of `max-width` and using `vw` instead of pixels: 

```suggestion
                width="30vw"
```

I tested this with various screen sizes, and ended up bumping it to 30% of the viewport width to make it look good on small screens.